### PR TITLE
Fix that some buildings are incorrectly available in TD missions

### LINKED ISSUE #12815: Some TD missions have buildable items that shouldn't be available
At least GDI 3 lets you build Guard Towers and Sandbags (shouldn't be available) and at least shows Concrete Walls (which you cannot build because the prerequisite isn't available).

There may be more cases like this.

It should be possible to fix most or even all of these before the next release.

### PR DESCRIPTION
**Pull request by reaperrr**
Closes #12815.

### CODE REVIEW

File: mods/cnc/maps/gdi03/rules.yaml [hunk 1]
```diff
@@ -37,10 +37,22 @@ GUN:
...
 
+GTWR:
+	Buildable:
+		Prerequisites: ~disabled
+
 ATWR:
 	Buildable:
 		Prerequisites: ~disabled
 
+SBAG:
```
Comment (abcdefg30) [inline mods/cnc/maps/gdi03/rules.yaml]: This conflicts with line 13.

**Comment by reaperrr** [comment]
Updated.

**Review by abcdefg30 — DISMISSED** [review]
You should also make the `Stealth Tank`, `Mobile S.A.M.` and `Artillery` in gdi05b unbuildable.

Also, if sandbags are unbuildable, why are chain link barriers available?
And, as there's nothing inside the support queue for the first (and second) mission, shouldn't we disable it entirely there (to avoid confusion)?

**Comment by reaperrr** [comment]
> You should also make the Stealth Tank, Mobile S.A.M. and Artillery in gdi05b unbuildable.

Done.

> Also, if sandbags are unbuildable, why are chain link barriers available?

I just forgot about them, fixed.

> And, as there's nothing inside the support queue for the first (and second) mission, shouldn't we disable it entirely there (to avoid confusion)?

In theory I guess that might be a good idea, but my attempt resulted in a crash I couldn't debug easily so I'd say we shouldn't risk that right now (assuming this still somehow makes it into this release).

**Comment by ltem** [comment]
...Review in progress...
I'm playing the maps right now, I hope I can give a review soon
Sandbags could be unlocked by capturing the enemy Construction Yard but I agree/prefer the changes of this PR.

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/57d0a267b6b3f0bd58f54f4696a6a03841ec9a20

**Pull request merged**